pinentry compilation failure

The only package that doesn't build on Fedora 30 beta is pinentry. If I ./configure in the pinentry using the host configuration, then it builds fine, but it doesn't build under the heads config.

First error is:
pinentry-gtk-2.c:28:28: fatal error: gdk/gdkkeysyms.h: No such file or directory
Followed by a whole set of similar issues once I solved that by hard-coding the gdk location  which natively would be provided by pkg-config --cflags gdk-2.0, I get other similar missing include files.

Should there be a gdk-2.0.pc file or similar in heads/install/lib/pkgconfig/ too?

--- a/modules/pinentry
+++ b/modules/pinentry
@@ -22,6 +22,8 @@ pinentry_configure := ./configure \
 	--disable-fallback-curses \
 	--disable-pinentry-curses \
 	--disable-pinentry-qt \
+  --disable-pinentry-gtk2 \
+  --disable-pinentry-gnome3 \
 	--disable-pinentry-fltk \
 	--disable-pinentry-emacs \
 	--disable-fallback-curses \

--- a/modules/pinentry
+++ b/modules/pinentry
@@ -22,8 +22,8 @@ pinentry_configure := ./configure \
 	--disable-fallback-curses \
 	--disable-pinentry-curses \
 	--disable-pinentry-qt \
-  --disable-pinentry-gtk2 \
-  --disable-pinentry-gnome3 \
+	--disable-pinentry-gtk2 \
+	--disable-pinentry-gnome3 \
 	--disable-pinentry-fltk \
 	--disable-pinentry-emacs \
 	--disable-fallback-curses \

